### PR TITLE
Introduce calculators namespace

### PR DESCRIPTION
**Pull request by andrewgarner**
Provides a foundation for further calculators.

### CODE REVIEW

**Comment by benbarnett** [comment]
:+1:

File: app/controllers/calculators/take_whole_pot_controller.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+module Calculators
+  class TakeWholePotController < CalculatorsController
+    def show
```
Comment (jiannis) [inline app/controllers/calculators/take_whole_pot_controller.rb]: Is it OK to override the [`show`](https://github.com/guidance-guarantee-programme/pension_guidance/blob/master/app/controllers/guides_controller.rb#L7) action defined in `GuidesController` that sets expiry cache headers?

File: app/controllers/calculators_controller.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+class CalculatorsController < GuidesController
```
Comment (jiannis) [inline app/controllers/calculators_controller.rb]: What does the `CalculatorsController` inherit from `GuidesController` ?

Comment (andrewgarner) [inline app/controllers/calculators_controller.rb]: Context of the Guide that the calculator is associated with. This allows us to display the title, breadcrumbs etc and keeps things consistent should the markdown change.

File: app/controllers/calculators/take_whole_pot_controller.rb (hunk 1 above)
Comment (andrewgarner) [inline app/controllers/calculators/take_whole_pot_controller.rb]: That occurred to me earlier.

We could set cache headers here too, however each result is likely to be a snowflake given different pot sizes and incomes values being passed in the querystring.

File: app/controllers/calculators_controller.rb (hunk 2 above)
Comment (jiannis) [inline app/controllers/calculators_controller.rb]: I see. Maybe at some point we could do: 

```ruby
class CalculatorsController < BaseController
  ...
end

class GuidesController < BaseController
  ...
end

class BaseController < ApplicationController
  before_action :find_page
  before_action :set_breadcrumbs

  private

  def find_page
    @page = ...
  end

  def set_breadcrumbs
    ...
  end
end

class Page
  attribute_reader :title, :description, :url, :slug
end
```

File: app/controllers/calculators/take_whole_pot_controller.rb (hunk 1 above)
Comment (jiannis) [inline app/controllers/calculators/take_whole_pot_controller.rb]: True. Probably the traffic on these calculators won't be extreme either.

**Pull request merged**